with-puppeteer-xvfb/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+DIR=$(cd $(dirname $0) && pwd)
+${DIR}/../test.sh
+WORK_DIR=$(mktemp --directory)
+#trap "rm -rf ${WORK_DIR}" EXIT
```

Comment:
Could be removed maybe?

Reply:
No, should be uncommented, forgot it thanks.

Reply:
Fixed.